Jenkinsfile

```diff
@@ -0,0 +1,143 @@
+  stages {
+    stage("Get requirements") {
+      steps {
+        sh "curl -O https://dl.google.com/go/${GO_TAR}"
+        sh "tar -xvf $GO_TAR"
+        sh "sudo mv go $GOROOT"
+        sh "sudo apt-get -y install e2fslibs-dev libfuse-dev libgpgme11-dev libdevmapper-dev libglib2.0-dev libprotobuf-dev"
+        sh "mkdir -p $GOPATH/src/github.com/intel"
+        sh "cp -rf ${env.WORKSPACE} $REPO_DIR"
+        dir(path: "$REPO_DIR") {
+          sh "go get -v golang.org/x/lint/golint"
+          sh "go get -v github.com/fzipp/gocyclo"
+        }
+        sh "mkdir -p ${GOPATH}/src/github.com/containers"
+        dir(path: "${GOPATH}/src/github.com/containers") {
+          sh "git clone --single-branch --depth 1 -b $BUILDAH_VERSION https://github.com/containers/buildah"
+        }
+        dir(path: "${GOPATH}/src/github.com/containers/buildah") {
+          sh 'make buildah TAGS=""'
+          sh "sudo cp buildah /usr/local/bin"
+          sh "sudo mkdir -p /etc/containers"
+          sh '''echo '[registries.search]' > registries.conf'''
+          sh '''echo 'registries = ["docker.io"]' >> registries.conf'''
+          sh "sudo mv registries.conf /etc/containers/registries.conf"
+          sh "sudo curl https://raw.githubusercontent.com/kubernetes-sigs/cri-o/$CRIO_VERSION/test/policy.json -o /etc/containers/policy.json"
+          sh "sudo curl -L https://github.com/opencontainers/runc/releases/download/$RUNC_VERSION/runc.amd64 -o /usr/bin/runc"
+          sh "sudo chmod +x /usr/bin/runc"
+        }
+     }
+   }
+   stage("make lint") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make lint"
+        }
+      }
+    }
+    stage("make format") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make format"
+        }
+      }
+    }
+    stage("make") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make"
+        }
+      }
+    }
+    stage("make vet") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make vet"
+        }
+      }
+    }
+    stage("make cyclomatic-check") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make cyclomatic-check"
+        }
+      }
+    }
+    stage("make test") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make test"
+        }
+      }
+    }
+    stage("make intel-fpga-admissionwebhook") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make intel-fpga-admissionwebhook"
+        }
+      }
+    }
+    stage("make intel-fpga-plugin") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make intel-fpga-plugin"
+        }
+      }
+    }
+    stage("make intel-gpu-plugin") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make intel-gpu-plugin"
+        }
+      }
+    }
+    stage("make intel-qat-plugin") {
+      steps {
+        dir(path: "$REPO_DIR") {
+          sh "make intel-qat-plugin"
+        }
+      }
+    }
+    stage("make images with buildah") {
```

Comment:
images without buildah are not built.
Also, use parallel stages for building something in variations (e.g. images with docker and images with buildah)
https://jenkins.io/blog/2017/09/25/declarative-1/

Reply:
Changes made, thanks for the review